app/utils/generateToken.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * On the subjust of generating random alphanumeric strings:
+ * A snippet of brilliance by stackoverflow user @doubletap, improved on by @amichair
+ * http://stackoverflow.com/questions/1349404/generate-a-string-of-5-random-characters-in-javascript
+ */
+export default function (n) {
+  var s = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
+  return (isNaN(n) && n >= 0) ? new Error('n must be of type Number and >= 0') : Array.apply(null, Array(n)).map(function () { return s.charAt(Math.floor(Math.random() * s.length)) }).join('')
```

Comment:
Just because this comes from SO doesn't mean we should throw all our style guidelines out the window. Does this line pass linting rules? It should have caused the build to fail. While some devs pride themselves on terseness and one-linerness, we do not. Please have this follow our normal style guidelines.